platforms/nuttx/nuttx-configs/omnibus-f4sd/nsh/defconfig

```diff
@@ -43,7 +43,7 @@ CONFIG_ARCH_MATH_H=y
 #
 # Debug Options
 #
-CONFIG_DEBUG_ALERT=y
```

Comment:
@davids5, i went ahead and pushed the results of `make menuconfig` w/o the uart4 changes for now, so we have some baseline for further work